theme/assets/js/ui/dialog.js

```diff
@@ -63,9 +65,10 @@ document.addEventListener('alpine:init', () => {
          * @param {Object} templateProps
          */
         openPrimaryDialog(...args) {
```

Comment:
Is there a reason this is an array? 

If not, and if there is time, I might refactor the args here into an options object with named params. I think of accessing an array by a specific index (Like we're doing on line 104 & 108) as a bad habit, because if the order changes, we'll have to update all the references

Reply:
And also when the array is accessed, it's not clear what's being accessed. You'd have to go and consult the jsdoc block and follow the logic path to figure out that `primaryDialogConfig[1]` is the dialog options object. I'd rather see `primaryDialogConfig.dialogOptions`

Reply:
https://github.com/square/brisk-theme/pull/32/commits/ba907c04eb0626be244b5ed86ee584c1b41b9c60